hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/endpoint/ObjectEndpoint.java

```diff
@@ -230,6 +230,14 @@ public Response put(
       }
       LOG.error("Exception occurred in PutObject", ex);
       throw ex;
+    } catch (OS3Exception ex) {
+      if (copyHeader != null) {
+        getMetrics().incCopyObjectFailure();
+      } else {
+        getMetrics().incCreateKeyFailure();
+      }
+      LOG.error("Exception occurred in PutObject", ex.getMessage());
```

Comment:
I don't think it is a good idea to log errors that can be triggered by user error. This should go into a dedicated access log https://issues.apache.org/jira/browse/HDDS-6559

Reply:
You are absolutely right I will replace Log.error() with Log.debug() !!

Reply:
@kerneltime should I replace the logger on line:231 as well?

Reply:
It is a good suggestion but we do not have an access log dedicated to S3 gateway, so we can leave it here for now!

Reply:
> we do not have an access log dedicated to S3 gateway

Now we do.

Reply:
@adoroszlai are you referring to audit log?

Reply:
Yes, audit log.

Reply:
Yeah just took a look, seems like the place to dump such errors but would need to be done only once for a request.